social-ui/src/MissedItCard.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import Meta from "./Meta";
+import missedArticles from './_data/missed-articles.json';
+
+
+class CardMissed extends Component {
+    constructor ()
+    {
+      super();
+      this.state={
+        missedArticles,
+      }
+    }
+
+
+    render() {
```

Comment:
This code for the article card is almost identical to the code in your render function of the ForYouCard and is a prime candidate for a reusable component.